src/webui/api/appcontroller.cpp

```diff
@@ -991,3 +992,19 @@ void AppController::networkInterfaceAddressListAction()
 
     setResult(addressList);
 }
+
+void AppController::supportedLanguagesAction()
```

Comment:
We no longer provide translation for third-party UI. And I strongly believe that the WebAPI itself shouldn't be locale-aware. So I wouldn't approve it to be provided in this way.
If it is supposed to be used by built-in WebUI only maybe just inject the generated locales in corresponding HTML file?